# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../agg/automake-fix.patch
# Copyright (C) 2014 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Index: Makefile.am
===================================================================
--- ./Makefile.am.orig
+++ ./Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = gpc src font_freetype font_win32_tt include examples
+SUBDIRS = src font_freetype font_win32_tt include examples
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libagg.pc
@@ -19,3 +19,4 @@ EXTRA_DIST = Makefile.AmigaOS \
 # M4 macro file for inclusion with autoconf
 m4datadir = $(datadir)/aclocal
 m4data_DATA = libagg.m4
+
Index: configure.in
===================================================================
--- ./configure.in.orig
+++ ./configure.in
@@ -1,14 +1,14 @@
-AC_INIT(src/agg_arc.cpp) # give me a source file, any source file...
+AC_INIT([agg], [2.5.0])
+AC_CONFIG_SRCDIR([src/agg_arc.cpp]) # give me a source file, any source file...
 AC_CANONICAL_TARGET
 AC_CONFIG_HEADERS(include/config.h)
-AM_INIT_AUTOMAKE(agg, 2.5.0)
+AM_INIT_AUTOMAKE([foreign])
 
 
 dnl Checks for programs.
 AC_PROG_CC
 AC_PROG_CXX
 AC_ISC_POSIX
-AM_C_PROTOTYPES
 if test "x$U" != "x"; then
   AC_MSG_ERROR(Compiler not ANSI compliant)
 fi
@@ -144,7 +144,6 @@ AC_SUBST(PREFERED_PLATFORM)
 AC_OUTPUT(
    Makefile
    libagg.pc
-   gpc/Makefile
    font_freetype/Makefile
    font_win32_tt/Makefile
    src/Makefile

